### DISCUSSION TITLE
change sidebar items

### DISCUSSION THREAD

**Discussion by Galleazzo** [ng-matero/ng-matero#641] [Q&A] [answered]
I want to change the sidebar for items in my use case. For example, I want to change the "design" to "registrations", and I want to change the link. I also want to delete other items from the list. Please
![image](https://github.com/ng-matero/ng-matero/assets/92340822/991a5b87-84bf-4a12-a81c-569afc14f0e3)

**Comment by nzbin** [comment, marked as answer]
I recommend using the `ng add ng-matero` to init the project. Please check this discussion https://github.com/ng-matero/ng-matero/discussions/552

**Reply by Galleazzo** [reply]
thank you very much. This solution solved my problem
